### DISCUSSION TITLE
Distributed architecture: after upgrade from 2.3.52 to 2.3.60 a grafana dashboards issue

### DISCUSSION THREAD

**Discussion by bioteseo** [Security-Onion-Solutions/securityonion#4860] [Unsupported Versions]
I have this distributed environment:
-1 manager
-8 search nodes
-4 forward nodes

I upgraded (by soup on manger) and the SOC->Grid displayed no problem. Then I modified the authentication to use Elstic authentication (as suggested here: https://docs.securityonion.net/en/2.3/so-elastic-auth.html). I did grafana changes as suggested here: https://docs.securityonion.net/en/2.3/grafana.html#grafana-changes-in-security-onion-2-3-60

Now, two forward nodes present a grafana dashboard old style (v 2.3.52). All the other nodes have a grafana dashboard new style (v 2.3.60). Anyone have a suggestion to correct/investigate this?

**Comment by TOoSmOotH** [comment]
Grid or Grafana? I am unaware where version is listed in grafana

**Reply by bioteseo** [reply]
I didn't mention a version visualization. I said grafana dashboard for two forward nodes are in old style like before to upgrade 2.3.60. In grid all nodes are 2.3.60.

**Reply by bioteseo** [reply]
Perhaps it's better if I share two images with you.
![newstylegrafanadashboards-after2 3 60upgrade](https://user-images.githubusercontent.com/316396/127000310-d24f6aae-7e32-4b31-9e6b-ea47da8a9cbb.png)
![oldstylegrafanadashboards-after2 3 60upgrade](https://user-images.githubusercontent.com/316396/127000338-0f16f1df-c99e-42b9-8e97-0a1627d62ba1.png)

In the oldstyle image there are two exclamation mark. The associated message is "InfluxDB Error: retention policy not found: autogen"
 
Thanks
